tyk-pro/templates/service-dash.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   ports:
   - port: {{ .Values.dash.service.port }}
     targetPort: {{ .Values.dash.containerPort }}
+    {{- if .Values.dash.service.nodePort }}
```

Comment:
`nodePort` is only valid if the `type` of Service is either `NodePort` or `LoadBalancer`. Perhaps worth specifying the condition accordingly to avoid cases of misconfiguration?